ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/netty/NettyPeerConnection.java

```diff
@@ -82,6 +87,16 @@ public void send(final Capability capability, final MessageData message) throws
                 + ") via cap "
                 + capability);
       }
+      outboundMessagesCounter
+          .labels(
+              capability.toString(),
+              subProtocol.messageName(capability.getVersion(), message.getCode()),
+              Integer.toString(message.getCode()))
+          .inc();
+    } else {
+      outboundMessagesCounter
+          .labels("null", Integer.toString(message.getCode()), Integer.toString(message.getCode()))
+          .inc();
```

Comment:
Messages with no capability would be one of the ones from `WireMessageCodes` so maybe could add a message name function there and provide the proper name rather than just the message code.

Reply:
Done.